WordPress/Classes/Services/PostRepository+Helpers.swift

```diff
@@ -4,7 +4,10 @@ import WordPressKit
 extension RemotePostCreateParameters {
     /// Initializes the parameters required to create the given post.
     init(post: AbstractPost) {
-        self.init(status: (post.status ?? .draft).rawValue)
+        self.init(
```

Comment:
Related to this change https://github.com/wordpress-mobile/WordPressKit-iOS/pull/757, I copied the solution from this PR https://github.com/wordpress-mobile/WordPress-iOS/pull/22848

Reply:
The CI continues to fail, it will be problematic until all the changes in https://github.com/wordpress-mobile/WordPress-iOS/pull/22849 are merged. 

Lets review this with failing CI, once we need to merge, we can coordinate with the other work to get this to pass.